### ISSUE TITLE
PL-117 ⁃ Change colour of copyright link in the footer to meet accessibility criteria

### ISSUE THREAD

**Issue by LaBustaffa** [comicrelief/pattern-lab#571]
Copyright link in the footer now doesn't meet accessibility criteria, as it doesn't contrast enough against the background. (See Retest report page 119)

It's enough to change it to white #ffffff to solve the issue. 

As we're there, might be a good idea to increase the space between it and the logo below to be the same with the line above. (See mockup attached)

![footer with clearer copy](https://user-images.githubusercontent.com/33353053/38419193-5d696690-3997-11e8-81d9-b6abcd3dafa2.png)

(Initially assigned to Adam for review)

**Comment by adamthomsCR** [comment]
I think that works to me. 
The SR is slightly different btw, as it has a purple background: https://puu.sh/zZqfv/560b8c6949.png

**Comment by deniselashlley** [comment]
changed the color in this ticket https://github.com/comicrelief/donate/issues/313

**Comment by CRSync** [comment]
➤ Unito Sync commented:

Transition made by Unito
